.github/workflows/add-release-reviewers.yml

```diff
@@ -9,29 +9,42 @@ permissions: {}
 
 jobs:
   add-reviewers:
-    if: ${{ github.event.label.name == 'release' }}
     # Tests that label is added on the PR
+    if: ${{ github.event.label.name == 'release' }}
     runs-on: ubuntu-latest
     permissions:
-      contents: read
+      contents: write
+      pull-requests: write
     steps:
       - id: get-members
-        uses: garnertb/get-team-members@0482f68c88601800f85145570ea93bf8fcfcf46f
-        with:
-          org: mui
-          team_slug: x
-          role: maintainer
-          token: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo ::setOutput name=data::$(\
+            gh api \
+              -H "Accept: application/vnd.github+json" \
+              -H "X-GitHub-Api-Version: 2022-11-28" \
+              /orgs/${{ORG}}/teams/${{TEAM_SLUG}}/members \
+              | jq 'reduce inputs as $i (.; . += $i)')
+        env:
+          ORG: ${{ github.event.organization }}
+          TEAM_SLUG: x
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      # parse return value and construct a string of login values
+      - id: construct-members
+        run: echo '::setOutput name=members::${{ join(fromJson(steps.get-members.outputs.data).*.login) }}'
```

Comment:
If you are not using data else where, you can just parse the value locally. Something like, though I didn't test it 😅 

```
DATA = $(gh api \
              -H "Accept: application/vnd.github+json" \
              -H "X-GitHub-Api-Version: 2022-11-28" \
              /orgs/${{ORG}}/teams/${{TEAM_SLUG}}/members \
              | jq 'reduce inputs as $i (.; . += $i)')

echo "members=${{ join(fromJson(DATA).*.login) }}" >> $GITHUB_OUTPUT
```

Reply:
Could be an option. I'd like to keep it as is for a future use of the data.

Reply:
did use the approach given by you, but with a slight variation 👍🏼